tutorials/easydeploy-vault/index.mdx

```diff
@@ -0,0 +1,175 @@
+content:
+  h1: Deploy HashiCorp Vault on Scaleway Kubernetes clusters using Easy Deploy
+  paragraph: Learn how to securely deploy and manage HashiCorp Vault on Scaleway Kubernetes clusters using the Easy Deploy feature. Follow our comprehensive guide for step-by-step instructions.
+tags: hashicorp vault kubernetes k8s easy deploy
+categories:
+  - containers
+dates:
+  validation: 2024-06-13
+  posted: 2024-06-13
+---
+
+HashiCorp Vault is an identity-based secrets and encryption management system.
+It provides encryption services that are gated by authentication and authorization methods to ensure secure, auditable and restricted access to secrets.
+Vault is used to secure, store and protect secrets and other sensitive data using a UI, CLI, or HTTP API.
+
+<Macro id="requirements" />
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- A valid [API key](/identity-and-access-management/iam/how-to/create-api-keys/)
+- Created a Scaleway Kubernetes [Kapsule](/containers/kubernetes/how-to/create-cluster/) or [Kosmos](/containers/kubernetes/how-to/create-kosmos-cluster/) cluster
+
+## Deploying the Vault application using Easy Deploy
+
+1. In the [Scaleway console](https://console.scaleway.com/), navigate to the **Kubernetes** section under **Containers**.
+2. Click the name of the cluster where you wish to deploy Grafana. The **Cluster Information** tab will display.
+3. Click the **Easy Deploy** tab. The application dashboard displays.
+4. Click **Deploy Application**. The application deployment wizard displays.
+5. Choose **Application Library** to see the list of available applications.
+6. Select the **Grafana** application.
+    <Message type="tip">
+        If you cannot find Vault on the first page, use the search bar or navigate through the library.
+    </Message>
+7. Optionally, customize the default configuration for Grafana using [Helm Charts](/tutorials/kubernetes-package-management-helm/). If you do not require any customized configuration you can skip this step.
+8. Enter a name and a Kubernetes namespace for your application. If no name is entered, Grafana will be installed in the default namespace of the cluster.
+9. Click **Deploy Application** to deploy Grafana on your Kubernetes cluster.
+
+## Initializing and unsealing Vault
+
+1. Check the status of your Vault using the `kubectl` command.
+    ```sh
+    kubectl get pods -l app.kubernetes.io/name=vault
+    ```
+    <Message type="tip">
+      If you choose anothter name for your Vault application ensure to replace the application name with the corresponding value.
+    </Message>
+
+2. Initialize Vault:
+    ```sh
+    kubectl exec -it vault-0 -- vault operator init
```

Comment:
Consider telling users that "vault-0" corresponds with the app name of this example: user should replace it with their own. I used "application-vault-0" when testing.